ts/screens/modal/RegionServiceModal.tsx

```diff
@@ -0,0 +1,191 @@
+import CookieManager, { Cookie } from "@react-native-community/cookies";
```

Comment:
Since it is not a modal but it used by a model could `RegionServiceWebViewPlayGround` a better name?